deployments/observability.mdx

```diff
@@ -104,3 +104,18 @@ Which would provide you with an interactive terminal.
 
 For more options please take a look at our
 [Readme](/reference/cli/#architect-exec-resource-flags----command).
+
+## Port forwarding
+
+Sometimes you want to access services running inside your cluster without exposing them publicly.
+The `port-forward` command will allow you access these services on your local machine.
+This can be useful for exposing non-http services for debugging.
```

Comment:
Change to:
"To access services running inside the cluster without exposing them publicly, you can use the `port-forward` command. This will allow you to access these services on your local machine and is useful for debugging non-HTTP services."